common/meta/types/proto/meta.proto

```diff
@@ -16,10 +16,12 @@ syntax = "proto3";
 
 package meta;
 
+import "request.proto";
+
 message Empty {}
-message GetRequest { string key = 1; }
+message RaftGetRequest { string key = 1; }
 
-message GetReply {
+message RaftGetReply {
```

Comment:
could this rename break the backwards compatibility of grpc api? q.q

Reply:
This patch is a breaking change by design.

Reply:
> could this rename break the backwards compatibility of the grpc api? q.q

ok, i will change it back.

Reply:
> This patch is a breaking change by design.

is this PR is mostly adding a new api? if so, i guess making break changes on the existed api may not be really required.

i think we can avoid the accidental breaking changes like fix naming styles if possible, but free to make break changes if it can not make the feature possible without a necessary architecture change 🤔